src/traces/sunburst/calc.js

```diff
@@ -128,7 +128,8 @@ exports.calc = function(gd, trace) {
         cd.unshift({
             hasMultipleRoots: true,
             id: dummyId,
-            pid: ''
+            pid: '',
+            label: ''
```

Comment:
@etpinard wondering are you OK with this change?

Reply:
What difference(s) does it make?

Reply:
The changing tests in https://github.com/plotly/plotly.js/commit/df5e1219dd2892b29d9c6a2fe5ec9f3dd3e4f2ae#diff-07da929bf23f8ed55f7b47eb51c0a739
I suggest we keep it blank so that the `getPath` helper function could see it as a blank string not an undefined:
https://github.com/plotly/plotly.js/blob/54360be4be39d7d77a8f88fb147909efa6b037d4/src/traces/sunburst/helpers.js#L165-L167

Reply:
If those `calcdata` diffs are the only difference, then that's not a problem :ok_hand: